meson.build

```diff
@@ -67,10 +67,15 @@ if host_machine.endian() == 'big'
 endif
 
 p11_kit = dependency('p11-kit-1', required: false)
-if p11_kit.found()
-  default_pkcs11_module = p11_kit.get_variable(pkgconfig: 'proxy_module')
-  conf.set_quoted('DEFAULT_PKCS11_MODULE', default_pkcs11_module)
+default_pkcs11_module = get_option('default_pkcs11_module')
+if default_pkcs11_module == 'no'
+  if p11_kit.found()
+    default_pkcs11_module = p11_kit.get_variable(pkgconfig: 'proxy_module')
+  else
+    error('default_pkcs11_module is empty')
```

Comment:
This will now fail to catch that default_pkcs11_module is empty because it has been moved inside a check that will not trigger if it is '' (or anything but 'no').

So the check that fails configuration if default_pkcs11_module is empty probably needs to be move out and done explicitly instead of via an else statement on p11_kit.found()